directory/cli/src/group.py

```diff
@@ -62,22 +70,27 @@ def show(group_name):
             raise click.ClickException(error)
 
 
+    @group.group(name='member', help='Handle group membership')
+    def member():
+        pass
+
     # TODO duplication
-    @group.command(name='add-member', help='Add user(s) to a group')
-    @click.argument('group_name')
-    @click.argument('users', nargs=-1, required=True)
-    def add_member(group_name, users):
-        _validate_blacklist_groups(group_name, users)
-        user_options = ['--users={}'.format(user) for user in users]
-        ipa_command = 'group-add-member'
-        args = [group_name] + user_options
-        try:
-            ipa_utils.ipa_run(ipa_command, args, error_in_stdout=True)
-            utils.display_success()
-        except IpaRunError:
-            _diagnose_member_command_error(group_name, users, add_command=True)
+    if utils.advanced_mode_enabled():
+        @member.command(name='add', help='Add user(s) to a group')
+        @click.argument('group_name')
+        @click.argument('users', nargs=-1, required=True)
+        def add_member(group_name, users):
+            _validate_blacklist_groups(group_name, users)
+            user_options = ['--users={}'.format(user) for user in users]
+            ipa_command = 'group-add-member'
+            args = [group_name] + user_options
+            try:
+                ipa_utils.ipa_run(ipa_command, args, error_in_stdout=True)
+                utils.display_success()
+            except IpaRunError:
+                _diagnose_member_command_error(group_name, users, add_command=True)
 
-    @group.command(name='remove-member', help='Remove user(s) from a group')
+    @member.command(name='remove', help='Remove user(s) from a group')
```

Comment:
Also: for consistency, it would probably make sense at some point to also change the `hostgroup add-member`/`hostgroup remove-member` commands to `hostgroup member add`/`hostgroup member remove`. This is probably beyond the scope of what you've been asked to do in Userware at the moment, though @ColonelPanicks may think it sounds worth doing sooner rather than later, but either way it would probably make sense to log an issue to remind us to change this at some point.

Reply:
Changes were made to `member add` after discussions with @ColonelPanicks. While `member remove` is less of an interactive process I could definitely change that to remain consistent. 

I do agree that at some point it would be good to change those `hostgroup` commands also.